Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -713,13 +713,15 @@ void WebChromeClient::unavailablePluginButtonClicked(Element& element, RenderEmb
 void WebChromeClient::mouseDidMoveOverElement(const HitTestResult& hitTestResult, unsigned modifierFlags, const String& toolTip, TextDirection)
 {
     RefPtr<API::Object> userData;
+    
```

Comment:
Super-nit: Don't need a blank line here.  it's not against style, but usually variables are grouped together instead of spaced.

Reply:
This doesn't need to be changed.  Added `merge-queue` keyword.